src/actions.rs

```diff
@@ -0,0 +1,886 @@
+//! Actions functionality for WebDriver.
+use crate::elements::Element;
+use std::fmt::Debug;
+use std::time::Duration;
+use webdriver::actions as WDActions;
+
+/// An action not associated with an input device (i.e. pause).
+#[derive(Debug, Clone)]
+pub enum NullAction {
```

Comment:
I suspect we'll want most of these to be `#[non_exhaustive]`. I don't think it costs us much, since these won't generally be matched on externally anyway, and it'll allow us to add new actions without doing a breaking release.